app/Providers/RouteServiceProvider.php

```diff
@@ -72,4 +73,12 @@ protected function mapApiRoutes()
              ->namespace($this->namespace)
              ->group(base_path('routes/api.php'));
     }
+
+    protected function mapStaffRoutes()
+    {
+        Route::prefix('')
+              ->middleware('web')
```

Comment:
@kai0310 

> なぜか下記エラーメッセージが出る。

こちら、スタッフモードのコントローラに `checkEnv` ミドルウェアが適用されていないのが原因です！

`checkEnv` ミドルウェア内部に、`DotenvService` の `getValue` メソッドを呼ぶ処理があるのですが、 `checkEnv` ミドルウェアが適用されていないので、 `getValue` メソッドが呼ばれず、PHPUnit が落ちてしまってます！

ここのテストがちょっとわかりづらいですね、、

-----

なので、PHPUnit が落ちないようにするには、以下のように修正すると良いかと思います！
(`RouteServiceProvider` の `mapWebRoutes` メソッドと同じ書き方にする)

```suggestion
              ->middleware(['web', 'install'])
```

Reply:
@SofPyon 
ありがとうございます。
仰る通りミドルウェアが適用されていなかったことを確認しましたので修正します。